utils/logger.py

```diff
@@ -0,0 +1,33 @@
+import logging
+import os
+from logging.handlers import TimedRotatingFileHandler
+
+log_directory = "logs"
+log_file = "app.log"
+
+if not os.path.exists(log_directory):
+    os.makedirs(log_directory)
+
+file_handler = TimedRotatingFileHandler(
+    filename=os.path.join(log_directory, log_file),
+    when="midnight",
+    interval=1,
+    backupCount=10,
+    encoding="utf-8"
+)
+file_handler.setLevel(logging.DEBUG)
+file_format = logging.Formatter('%(asctime)s - %(name)s - %(levelname)s - %(message)s')
+file_handler.setFormatter(file_format)
+
+console_handler = logging.StreamHandler()
+console_handler.setLevel(logging.DEBUG)
```

Comment:
lets set console logger to INFO, for users. Devs can switch to debug manually or look at the file logger.

Reply:
Done